### PR TITLE
Side nav update

### PR DESCRIPTION
**Pull request by KevinKrames**
Update side nav aligning and added records button

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/3See/etect-app/pull/32" target="_blank">ReviewNinja

File: modules/core/client/views/sidebar.client.view.html [hunk 1]
```diff
@@ -17,27 +17,78 @@
                 <li>Current Study</li>
                 <li>
                     <button type="button" class="btn btn-transparent btn-block" data-toggle="button" ui-sref="home">
-                        <span class="glyphicon glyphicon-menu-hamburger"></span>Dashboard
+                        <div class="row">
+                            <div class="col-xs-1 col-sm-1 col-md-1 col-lg-1">
+                                <span class="glyphicon glyphicon-menu-hamburger"></span>
+                            </div>
+                            <div class="col-xs-10 col-sm-10 col-md-10 col-lg-10">
+                                <span style="float:left;">Dashboard</span>
```
Comment (Blistin) [inline modules/core/client/views/sidebar.client.view.html]: !fix For bootstrap, float can be used as pull, do a class="pull-left" for each of the times you used float

**Comment by Blistin** [comment]
@KevinKrames make sure you pull the update down before you go through with the fixes

File: modules/core/client/views/sidebar.client.view.html (hunk 1 above)
Comment (csturco0331) [inline modules/core/client/views/sidebar.client.view.html]: !fixed

**Pull request merged**